code/datums/elements/light_eater.dm

```diff
@@ -51,7 +51,9 @@
 
 	. = 0
 	for(var/morsel in buffet)
-		. += devour(morsel, eater)
+		// You can't eat space, silly
+		if(!istype(morsel, /turf/open/space))
```

Comment:
don't we have an `isspaceturf` or something check

Reply:
Maybe but doesn't really matter, the problem isn't exclusive to space tiles and it's a whole bigger thing, working on it